body-tracking-samples/sample_helper_libs/sample_shared_lib/DigitalSignalProcessing.cpp

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "DigitalSignalProcessing.h"
```

Comment:
I need some background info. Why is the code needed now? Why original sample code can be built without them?

Reply:
The update makes sense to me.